src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/TagHelperCompletionProvider.cs

```diff
@@ -8,30 +8,31 @@
 using Microsoft.AspNetCore.Razor.Language;
 using Microsoft.AspNetCore.Razor.Language.Legacy;
 using Microsoft.AspNetCore.Razor.Language.Syntax;
+using Microsoft.CodeAnalysis.Razor.Completion;
 using Microsoft.VisualStudio.Editor.Razor;
-using OmniSharp.Extensions.LanguageServer.Protocol.Models;
-using RazorTagHelperCompletionService = Microsoft.VisualStudio.Editor.Razor.TagHelperCompletionService;
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Completion
 {
-    internal class DefaultTagHelperCompletionService : TagHelperCompletionService
+    internal class TagHelperCompletionProvider : RazorCompletionItemProvider
     {
-        private static readonly Container<string> NoCommitCharacters = new Container<string>();
-        private static readonly Container<string> AttributeCommitCharacters = new Container<string>("=", " ");
-        private static readonly Container<string> MinimizedAttributeCommitCharacters = new Container<string>("=");
-        private static readonly Container<string> ElementCommitCharacters = new Container<string>(" ", ">");
+        // Internal for testing
+        internal static readonly IReadOnlyCollection<string> MinimizedAttributeCommitCharacters = new List<string> { "=", " " };
```

Comment:
👏

Reply:
Ah ya forgot to call that out. Found that the names were off. Was an unrelated fix that threw me for a loop when I was writing tests 😄